packages/block-editor/src/hooks/use-bindings-attributes.js

```diff
@@ -97,6 +97,8 @@ export const withBlockBindingSupport = createHigherOrderComponent(
 			unlock( select( blocksStore ) ).getAllBlockBindingsSources()
 		);
 		const { name, clientId, context } = props;
+		const hasDefaultBinding =
+			props.attributes.metadata?.bindings?.[ DEFAULT_ATTRIBUTE ];
```

Comment:
Instead of just checking if it has the `__default` attribute, maybe we can check that it is `core/pattern-overrides` as we are doing [here](https://github.com/WordPress/gutenberg/blob/95999a97132560b801e9e18d28d33a87ff313e4e/packages/block-editor/src/hooks/use-bindings-attributes.js#L48-L50)? If we do that, we might want to change the name of the variabel.

If we decide to support `__default` in any source in the future, I am not sure if skipping `setAttributes` for any attribute would be okay for all of them. It seems specific to pattern overrides to me.

Additionally, adding the `__default` property pointing to any other source, currently not supported, would skip `setAttributes` as well.

Reply:
Sounds good. I've made that change 👍 

Ultimately I'm not sure if this PR is the right fix, but it's a small change, and I can't think of many other options.